build.sbt

```diff
@@ -1,19 +1,24 @@
 import Dependencies._
 
-lazy val defaultSettings = Seq(
-  name := "cromwell_pipeline",
-  version := "0.1",
-  scalaVersion := "2.12.9",
-  defaultScalacOptions)
+ThisBuild / version := "0.1"
+ThisBuild / scalaVersion := "2.12.9"
+ThisBuild / scalacOptions := Seq(
+    "-encoding", "utf8",
+    "-deprecation",
+    "-Xfatal-warnings"
+)
+
+lazy val root = (project in file("."))
+    .settings(name := "Cromwell pipeline")
+    .aggregate(portal)
 
-lazy val cromwell_pipeline = (project in file("."))
-    .settings(
-      defaultSettings, 
-      libraryDependencies ++= akkaDependencies ++ dbDependencies ++ testDependencies
-    )
 
-lazy val defaultScalacOptions = scalacOptions ++= Seq(
-  "-encoding", "utf8",
-  "-deprecation",
-  "-Xfatal-warnings"
-)
\ No newline at end of file
+lazy val datasource = project.settings(
+    name := "Datasource",
+    libraryDependencies ++= dbDependencies)
+
+lazy val portal = project
```

Comment:
I'm really confused by the naming

Reply:
The current naming works fine for me and this is not a surprise :)

If you would like, we can discuss another naming together.